MountSionCBSSecondary/SchoolWebsite
# Evanhearne1 adding content to anibullying policy

# Description

Adjusted policy.html in /pages/about/ to account for new content added to antibullying.html in /pages/about/policies/. Added edited content from Mount Sion Website Google Drive Anti Bullying Document. 

## What

Anti Bullying Section of Policies Part of Website now has content.
Anti Bullying Read More Part of Policies Part of Website now has content

## Why

To add more policies to the website. It allows more content that's useful to be added to the website.

## How

Changes were made via Sublime Text 3. Changes were made by me (evanhearne).

## Verification Steps

Clone evanhearne1-adding-content-to-antibullying-policy from evanhearne's fork
Initialize a localhost https-server in the root directory of the clone
Open localhost server and go to About --> Policies
Go down to Antibullying and ensure blurb is there
Open the Read More and ensure that document is displayed correct 
Give suggestions of changes or just approve this pull request :)


# Checklist:
